src/components/plots/abstractStackedPlot.ts

```diff
@@ -122,7 +122,7 @@ export module Plot {
 
           var value = valueAccessor(datum, datumIndex, dataset.metadata(), plotMetadata);
           var offset: number;
-          if (value === 0) {
+          if (value === 0 || value === "0") {
```

Comment:
Well, if we don't this is not the place to begin doing that. Shall I file a new issue about this? Keep in mind that `d3` scales accept strings as arguments, np